cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4MeasureProcessor.java

```diff
@@ -97,6 +97,12 @@ protected MeasureReport evaluateMeasure(
 
         var subjects =
                 subjectProvider.getSubjects(actualRepo, evalType, subjectIds).collect(Collectors.toList());
+        /*
```

Comment:
This comment block is no longer needed, yes?

Reply:
removed